pkg/machines/helpers.es6

```diff
@@ -192,7 +192,7 @@ export function fileDownload ({ data, fileName = 'myFile.dat', mimeType = 'appli
     window.setTimeout(() => { // give phantomJS time ...
         logDebug('removing temporary A.HREF for filedownload');
         document.body.removeChild(a);
-    }, 500);
+    }, 5000);
```

Comment:
Thanks for pointing out! Indeed there is a little flickering spot on the bottom of the page. I tried to add `a.style.visibility = 'hidden'`, but that only hides the link itself. But since it has no content, it is already not visible (you can test with adding `a.appendChild(document.createTextNode("blablabla"))`).

So that little speck of ugliness is not the `<a>` link itself, and it also isn't sensitive to 500 vs. 5000 ms, it just appears there for a split second either way. So I think this can be fixed independently.

@mareklibra, I retried the tests. Once they get green, could you land this? Thanks for the review!